Source/WebCore/platform/cocoa/VideoFullscreenModelVideoElement.h

```diff
@@ -90,6 +86,20 @@ class VideoFullscreenModelVideoElement : public VideoFullscreenModel, public Eve
     WEBCORE_EXPORT VideoFullscreenModelVideoElement();
 
 private:
+    class VideoListener final : public EventListener {
+    public:
+        static Ref<VideoListener> create(VideoFullscreenModelVideoElement* parent)
+        {
+            return adoptRef(*new VideoListener(parent));
+        }
+        void handleEvent(WebCore::ScriptExecutionContext&, WebCore::Event&) override;
+        bool operator==(const EventListener& rhs) const override { return static_cast<const WebCore::EventListener*>(this) == &rhs; }
```

Comment:
Is `return this == &rhs;` not working?
Maybe this implementation should be the default one.

Reply:
I copied/paste the code, not looking much. the method used to inherit from two classes. I wonder why this even needs an override tbh